src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
```

Comment:
The variable name `resultObject` is not descriptive enough. Consider using a name that reflects the data it holds, such as `styleObject` or `cssProperties`.